scripts/cleanup_dp.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+set -o errexit
+
+mkdir ~/.aws; touch ~/.aws/config
+
+echo "[default]
+aws_access_key_id = $AWS_ACCESS_KEY_ID
+aws_secret_access_key = $AWS_SECRET_ACCESS_KEY" > ~/.aws/credentials
+
+echo "[default]
+[profile eksci]
+role_arn= $AWS_ROLE_ARN
+output = json
+region=ap-south-1
+source_profile = default" > ~/.aws/config
+
+export region=ap-south-1
+export cluster_name=uat-cluster
+
+echo "Region: $region"
+echo "Cluster name: $cluster_name"
+
+sts_output=$(aws sts assume-role --role-arn env.AWS_ROLE_ARN --role-session-name ekscisession)
```

Comment:
Did you mean `$AWS_ROLE_ARN` here?

```suggestion
sts_output=$(aws sts assume-role --role-arn "$AWS_ROLE_ARN" --role-session-name dp-cleanup-script)
```